src/components/m-dropdown/mixins/dropdown.scss

```diff
@@ -156,7 +156,7 @@
 }
 
 @mixin dropdown-icon($size: $dropdown__icon-size, $right: $dropdown__frame-x-padding, $color: $dropdown__icon-color) {
-  @include hardware-accelerate();
+  // @include hardware-accelerate(); Chrome has an issue
```

Comment:
I really want them for mobile, though I'm fine commenting them out right now

Reply:
agree. But curretnly works for mobile as well so its okish. :(

Reply:
so can i merge?

Reply:
sure merge it, but I'm really eager and considered to find a cross-browser solution here...

Reply:
:) ok
just keep in mind that css 3d transform is also still an experimental feature and could bring other problems in future. I have used it as well lots of time and like the idea of using the gpu, but please consider this as not high prio.

May be worst reading this article here: https://www.smashingmagazine.com/2016/12/gpu-animation-doing-it-right/

Reply:
same as WC at the end ;)